config/initializers/airbrake.rb

```diff
@@ -1,7 +1,17 @@
-if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
-  require 'airbrake'
+require 'airbrake'
 
+if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
   Airbrake.configure do |config|
     config.api_key = ENV['AIRBRAKE_API_KEY']
   end
+else
+  module Airbrake
```

Comment:
I'm overriding the Airbrake module instead of creating a new one because there were some functions setting the logger and calling other public methods if it was defined. So I decided to just override the particular methods that should output to the rails logger, and leave the rest of the methods as-is.